torch-sys/build.rs

```diff
@@ -118,19 +163,11 @@ fn prepare_libtorch_dir() -> PathBuf {
                     }
                 ),
                 "macos" => {
-                    if let Ok(arch) = env::var("CARGO_CFG_TARGET_ARCH") {
-                        if arch.as_str() == "aarch64" {
-                            panic!("Pre-built version of libtorch for apple silicon are not available.
-                            You can install torch manually following the indications from https://github.com/LaurentMazare/tch-rs/issues/629
-                            pip3 install torch=={TORCH_VERSION}
-
-                            Then update the following environment variables:
-                            export LIBTORCH=$(python3 -c 'import torch; from pathlib import Path; print(Path(torch.__file__).parent)')
-                            export DYLD_LIBRARY_PATH=${{LIBTORCH}}/lib
-                            ");
-                        }
+                    if env::var("CARGO_CFG_TARGET_ARCH") == Ok(String::from("aarch64")) {
+                        get_pypi_wheel_url_for_aarch64_macosx().expect("Failed to python wheel url from pypi")
```

Comment:
Could we put back the original error message in the expect here, this way users might have some workable failover to try and would know which github issue to look at.